apps/condo/domains/ticket/utils/clientSchema/Renders.tsx

```diff
@@ -186,4 +188,27 @@ export const getTicketClientNameRender = (search: FilterValue) => {
 
         return getTableCellRenderer(search, false, null, null, null, clientName)(trimmedClientName)
     }
+}
+
+const HINT_STYLES: CSSProperties = { maxHeight: '6.5em', maxWidth: '300px', overflow: 'hidden', wordBreak: 'break-word', whiteSpace: 'inherit' }
+
+export const getTicketHintRender = (search: FilterValue) => {
```

Comment:
According to implementation of this function, it renders html content.
Maybe to create a React component `HtmlContent` in `common` domain?

Reply:
Added `HtmlContent` in common domain and reuse it here